atomic_reactor/tasks/common.py

```diff
@@ -85,3 +86,6 @@ def __init__(self, params: TaskParams):
     @abc.abstractmethod
     def execute(self):
         """Execute this task."""
+
+    def _get_build_dir(self) -> RootBuildDir:
+        """Create a build directory object for this container build task."""
```

Comment:
Should this be an abstract method?

Reply:
The clone task does not require a build dir.

Reply:
Hmm, then maybe the method doesn't need to be in the Task class at all?

Reply:
Oh, good catch! :sweat_smile: